docs/operations.md

```diff
@@ -141,62 +123,6 @@ stays in the database:
 
 **Default**: `30` (days)
 
-### Always keep scavenged
-
-Scavenging aims to save disk space. Therefore, if the scavenging process finds out that the new chunk is for
-some reason larger or has the same size as the old chunk, it won't replace the old chunk because there's no
-disk space to save.
-
-Such behaviour, however, is not always desirable. For example, you might want to be sure that events from
-deleted streams are removed from the disk. It is especially relevant in the context of personal data deletion.
-
-When the `AlwaysKeepScavenged` option is set to `true`, EventStoreDB would replace the old chunk with the new
-one unconditionally, giving you guarantee that all the deleted events in the scavenged chunk actually
-disappear.
-
-| Format               | Syntax                             |
-|:---------------------|:-----------------------------------|
-| Command line         | `--always-keep-scavenged`          |
```

Comment:
the options still shows up in `--help` , should be classified as unused here ?

Reply:
we'll probably remove them from the code

Reply:
done here https://github.com/EventStore/EventStore/pull/3778